extension/src/plots/model/custom.ts

```diff
@@ -35,23 +19,17 @@ export const getFullValuePath = (type: string, columnPath: string) =>
 export const cleanupOldOrderValue = (
   value: { metric: string; param: string } | CustomPlotsOrderValue
 ): CustomPlotsOrderValue => ({
-  // previous column paths have the "TYPE:" prefix
   metric: removeColumnTypeFromPath(value.metric, ColumnType.METRICS),
-  param: removeColumnTypeFromPath(value.param, ColumnType.PARAMS),
-  // previous values didn't have a type
-  type: (value as CustomPlotsOrderValue).type || CustomPlotType.METRIC_VS_PARAM
```

Comment:
Are we sure about wanting to delete the `type` in workspace state values? If we want to add a different type of custom plot in the future, we're going to have to adjust our state again, adding the type back. Might be easier to just keep it even if it's not being used in the code currently 🤔

Reply:
YAGNI